pkg/diffs/diffs.go

```diff
@@ -72,6 +73,25 @@ func GetChangedCiopConfigs(masterConfig, prConfig config.DataByFilename, logger
 			}
 		}
 
+		oldResources := oldConfig.Configuration.Resources
+		newResources := newConfig.Configuration.Resources
+
+		for as, resource := range newResources {
+			if as == "*" && !equality.Semantic.DeepEqual(oldResources["*"], resource) {
+				for testName := range newTests {
+					jobs.Insert(testName)
```

Comment:
Changing `"*"` will only affect tests that otherwise do not have their own specific config, so this is still greedy